### ISSUE TITLE
Rush timer options

### ISSUE THREAD

**Issue by Novaras** [HW-PlayersPatch/Development#345]
This is a 2-minute job, but the issue is I don't want to write locale stuff.

https://discordapp.com/channels/128196211969753088/135543171768647680/706709240739528714

If you don't mind, I'll just append english options to the list in `scripts/deathmatch.lua`:

```lua
choices =
{
    -- "$5455", "0",
    -- "$2992", "5",
    -- "$2993", "10",
    -- "$3002", "15",
    -- adding these:
    "None", "0",
    "5 Minutes", "5",
    "10 Minutes", "10",
    "15 Minutes", "15",
    "20 Minutes", "20",
    "25 Minutes", "25",
    "30 Minutes", "30",
    "60 Minutes", "60",
},
```

**Comment by cloaked2222** [comment]
Generally when adding new options I consider a few things:
1. Is this useful to a significant number of people?
2. Does this negatively impact anyone?
3. Is this a quality addition that Relic/GBX would have added if they had the time?
4. Can this be implemented well?
5. Etc.

1. Maybe useful to a couple people.
2. No
3. No. A 15minute rush timer is plenty, most rts games are over in 15minutes. If you just want to practice builds without the AI attacking you, you can play vs an observer offline.
4. Not ideal, would likely need to be in English only. Not sure if there's audio that's played for this in addition to the text.

Bottom line, there are tones of options that could be added, but not looking to add the kitchen sink. I feel we already have too many options in the game setup that are not needed as it is. For example, we could easily add 'Max' unit caps, allowing for say 256 of any unit. But this is pretty meh from a balance/quality game studio perspective. Additionally our time is limited with a long list of things to work on, and even simple 2 minute change jobs like this require a longer testing/validation/change log/readme time effort tail.
